test/integration-test.js

```diff
@@ -32,19 +32,20 @@ describe('transport-level client/server integration', () => {
     let p = new Promise(res => {
       httpServer = server.app.listen(PORT, () => res());
     });
-    p.then(() => { return jose.newkey(); })
-      .then(k => {
-        let client = new TransportClient({accountKey: k});
-        return client.post(url, query);
-      })
-      .then(body => {
-        assert.isTrue(gotPOST);
-        assert.deepEqual(body, result);
-      })
-      .then(() => {
-        httpServer.close();
-        done();
-      })
-      .catch(done);
-  });
+    p
```

Comment:
This seems like the main weirdness here.  Could we make the rule that mid-line `.then` doesn't need anything special in terms of indent?